app/models/concerns/waste_carriers_engine/can_use_edit_registration_workflow.rb

```diff
@@ -27,10 +167,102 @@ module CanUseEditRegistrationWorkflow
         end
 
         event :back do
+          # Registration and account
+          transitions from: :cbd_type_form,
+                      to: :edit_form
+
+          # Business details
+          transitions from: :company_name_form,
+                      to: :edit_form
+
+          transitions from: :main_people_form,
+                      to: :edit_form
+
+          # Company address
+          transitions from: :company_postcode_form,
+                      to: :edit_form
+
+          transitions from: :company_address_form,
+                      to: :company_postcode_form
+
+          transitions from: :company_address_manual_form,
+                      to: :edit_form,
+                      if: :based_overseas?
+
+          transitions from: :company_address_manual_form,
+                      to: :company_postcode_form
+
+          # Contact details
+          transitions from: :contact_name_form,
+                      to: :edit_form
+
+          transitions from: :contact_phone_form,
+                      to: :edit_form
+
+          transitions from: :contact_email_form,
+                      to: :edit_form
+
+          # Contact address
+          transitions from: :contact_postcode_form,
+                      to: :edit_form
+
+          transitions from: :contact_address_form,
+                      to: :contact_postcode_form
+
+          transitions from: :contact_address_manual_form,
+                      to: :edit_form,
+                      if: :based_overseas?
+
+          transitions from: :contact_address_manual_form,
+                      to: :contact_postcode_form
+
+          # Location
+          transitions from: :location_form,
+                      to: :edit_form
+
+          # Complete an edit
           transitions from: :declaration_form,
                       to: :edit_form
         end
+
+        event :skip_to_manual_address do
+          transitions from: :company_postcode_form,
+                      to: :company_address_manual_form
+
+          transitions from: :company_address_form,
+                      to: :company_address_manual_form
+
+          transitions from: :contact_postcode_form,
+                      to: :contact_address_manual_form
+
+          transitions from: :contact_address_form,
+                      to: :contact_address_manual_form
+        end
       end
+      # rubocop:enable Metrics/BlockLength
+    end
+
+    private
+
+    def based_overseas?
+      overseas?
+    end
+
+    def registered_address_was_manually_entered?
+      return unless registered_address
+
+      registered_address.manually_entered?
```

Comment:
Seems like this is missing test coverage :)

Reply:
I think this was a copy-paste goof - we don't actually use them for anything! So I've ripped 'em out.